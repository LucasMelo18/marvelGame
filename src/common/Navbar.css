.Nav{
  align-items: center;
  background-color: #22262a;
  display: flex;
  justify-content: space-between;
  height: 50px;
  width: 100%;
}

.Nav section:nth-child(2){
  width: 50%;
  display: flex;
  justify-content: space-around;
  padding-right: 100px;
}

#mainNav a{
  color: white;
  cursor: pointer;
  font-family: DIN Next W01 Regular,Trebuchet MS,Helvetica,Arial,sans-serif;
  font-size: 13px;
  font-weight: 700;
  letter-spacing: 1px;
  line-height: 16px;
  text-transform: uppercase;
}


#mainNav a:hover{
  text-decoration: underline;
  text-decoration-color: #ee4326;
}

.Nav .logoQuiz{
  width: 100px;
  height: auto;
  margin: 0 50px;
}



@media screen and (max-width: 768px){
  .Nav{
    flex-direction: column;
    justify-content: space-around;
    height: 130px;
  }
  .Nav section:nth-child(2){
    justify-content: space-between;
    width: 100%;
    padding-left: 20px;
    padding-right: 20px;
  }

}