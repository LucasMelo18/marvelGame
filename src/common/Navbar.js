import React, { Component } from 'react';
import './Navbar.css';
import logoMarvelNav from '../assets/images/marvelLogo.png';

class Nav extends Component {
  render() {
    return (
      <div id="mainNav">
        <nav className="Nav">
          <section>
            <img src={logoMarvelNav} alt="Logo Marvel" className='logoQuiz'/> 
          </section>
          <section id="linksDesktop">
            <a rel="noopener noreferrer" href="https://github.com/LucasMelo18" target="_blank">Github</a>
            <a rel="noopener noreferrer" href="https://www.linkedin.com/in/lucasmelo18/" target="_blank">Linkedin</a>
            <a rel="noopener noreferrer" href="https://api.whatsapp.com/send?phone=5511985861957&text=Ola" target="_blank">Whatsapp</a>
          </section>

        </nav>

      </div>
    );
  }
}

export default Nav;
