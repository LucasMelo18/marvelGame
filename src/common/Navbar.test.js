import React from 'react';
import renderer from 'react-test-renderer';

import Navbar from './Navbar';

describe('Navbar', () => {
  var subject;
  var tree;

  beforeEach(() => {
    subject = renderer.create(<Navbar />);
    tree = subject.toJSON();
  });

  it('is expected to match the snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
