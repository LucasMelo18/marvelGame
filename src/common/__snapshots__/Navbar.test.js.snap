// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar is expected to match the snapshot 1`] = `
<div
  id="mainNav"
>
  <nav
    className="Nav"
  >
    <section>
      <img
        alt="Logo Marvel"
        className="logoQuiz"
        src="marvelLogo.png"
      />
    </section>
    <section
      id="linksDesktop"
    >
      <a
        href="https://github.com/LucasMelo18"
        rel="noopener noreferrer"
        target="_blank"
      >
        Github
      </a>
      <a
        href="https://www.linkedin.com/in/lucasmelo18/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Linkedin
      </a>
      <a
        href="https://api.whatsapp.com/send?phone=5511985861957&text=Ola"
        rel="noopener noreferrer"
        target="_blank"
      >
        Whatsapp
      </a>
    </section>
  </nav>
</div>
`;
